src/light-box/share.js

```diff
@@ -0,0 +1,12 @@
+/* @flow */
+import { Share } from 'react-native';
+
+import { BRAND_COLOR } from '../styles';
+
+export default (url: string) => {
+  const shareOptions = {
+    message: url,
+    title: 'Shared using Zulip',
```

Comment:
Can be added to [src/i18n/translations/ ](https://github.com/zulip/zulip-mobile/tree/master/src/i18n/translations)